ui/apps/platform/src/Containers/Clusters/Components/ClusterStatus.tsx

```diff
@@ -21,13 +23,58 @@ function ClusterStatus({ healthStatus, isList = false }: ClusterStatusProps): Re
     const { overallHealthStatus } = healthStatus;
     const { Icon, bgColor, fgColor } = healthStatusStyles[overallHealthStatus];
     const icon = <Icon className={`${isList ? 'inline' : ''} h-4 w-4`} />;
+
+    const unhealthyClusterDetailAvailable = overallHealthStatus === 'UNHEALTHY';
+    const bodyContent = (
+        <Button
+            component="a"
+            href="https://docs.openshift.com/acs/installing/installing_cloud_ocp/prerequisites-cloud-ocp.html#collector-prerequisites_prerequisites-cloud-ocp"
```

Comment:
Is this the final link? I'm not sure the troubleshooting document is available in the docs just yet, and I was under the impression that we'd link directly there.

Reply:
@lmaynard88 
Please confirm the proper link.

Reply:
@lmaynard88 
Can you give me the correct URL for this change now?

Reply:
@lmaynard88 I have changed the URL and verified that it is used. (Page doesn't exist yet.)
![Screen Shot 2023-01-30 at 11 24 08 AM](https://user-images.githubusercontent.com/715729/215535100-692a19ec-8c7b-4484-9620-7ecbb0434fcd.png)